### PR TITLE
Aml 3276 update code error messages

### PR DESCRIPTION
**Pull request by Winsto**
Updated contents.

### CODE REVIEW

File: src/SFA.DAS.EmployerUsers.Application/Commands/ActivateUser/ActivateUserCommandValidator.cs [hunk 1]
```diff
@@ -8,6 +8,9 @@ namespace SFA.DAS.EmployerUsers.Application.Commands.ActivateUser
 {
     public class ActivateUserCommandValidator : IValidator<ActivateUserCommand>
     {
+        private string _invalidCodeMessage = "Code is not recognised";
```
Comment (soharadesign) [inline src/SFA.DAS.EmployerUsers.Application/Commands/ActivateUser/ActivateUserCommandValidator.cs]: add a const keyword and update the name as appropriate

File: src/SFA.DAS.EmployerUsers.Application/Commands/ActivateUser/ActivateUserCommandValidator.cs [hunk 2]
```diff
@@ -8,6 +8,9 @@ namespace SFA.DAS.EmployerUsers.Application.Commands.ActivateUser
 {
     public class ActivateUserCommandValidator : IValidator<ActivateUserCommand>
     {
+        private string _invalidCodeMessage = "Code is not recognised";
+        private string _missingCodeMessage = "Enter the code we emailed you";
```
Comment (soharadesign) [inline src/SFA.DAS.EmployerUsers.Application/Commands/ActivateUser/ActivateUserCommandValidator.cs]: add a const keyword and update the name as appropriate

File: src/SFA.DAS.EmployerUsers.Application.UnitTests/CommandsTests/ActivateUserTests/ActivateUserCommandValidatorTests/WhenValidatingTheRequest.cs [hunk 3]
```diff
@@ -13,6 +13,9 @@ namespace SFA.DAS.EmployerUsers.Application.UnitTests.CommandsTests.ActivateUser
     public class WhenValidatingTheRequest
     {
         private ActivateUserCommandValidator _activateUserCommandValidator;
+        private string _invalidCodeMessage = "Code is not recognised";
```
Comment (soharadesign) [inline src/SFA.DAS.EmployerUsers.Application.UnitTests/CommandsTests/ActivateUserTests/ActivateUserCommandValidatorTests/WhenValidatingTheRequest.cs]: add a const keyword and update the name as appropriate

File: src/SFA.DAS.EmployerUsers.Application.UnitTests/CommandsTests/ActivateUserTests/ActivateUserCommandValidatorTests/WhenValidatingTheRequest.cs [hunk 4]
```diff
@@ -13,6 +13,9 @@ namespace SFA.DAS.EmployerUsers.Application.UnitTests.CommandsTests.ActivateUser
...
+        private string _missingCodeMessage = "Enter the code we emailed you";
```
Comment (soharadesign) [inline src/SFA.DAS.EmployerUsers.Application.UnitTests/CommandsTests/ActivateUserTests/ActivateUserCommandValidatorTests/WhenValidatingTheRequest.cs]: add a const keyword and update the name as appropriate

File: src/SFA.DAS.EmployerUsers.Application/Commands/ActivateUser/ActivateUserCommandValidator.cs (hunk 1 above)
Comment (Winsto) [reply src/SFA.DAS.EmployerUsers.Application/Commands/ActivateUser/ActivateUserCommandValidator.cs]: I am generally wary of `const` for things that are not really constant, due to nature of consts. But in this case we would always rebuild whole thing if we changed that string anyhow, so happy to change it.

File: src/SFA.DAS.EmployerUsers.Application/Commands/ActivateUser/ActivateUserCommandValidator.cs (hunk 2 above)
Comment (Winsto) [reply src/SFA.DAS.EmployerUsers.Application/Commands/ActivateUser/ActivateUserCommandValidator.cs]: As per comment on other; changed.

**Pull request merged**